data_files/IBMZ-container-registry.json

```diff
@@ -0,0 +1,672 @@
+[
+  {
+    "packageName": "alpine",
+    "version": "3.10",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.11",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.12",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.13",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.6",
+    "description": "Alpine Linux is a Linux distribution."
+  },
+  {
+    "packageName": "alpine",
+    "version": "3.7",
+    "description": "Alpine Linux is a Linux distribution."
```

Comment:
Is the file intended to have multiple versions of one package?Shouldn't it be the latest version per package?

Reply:
Yes, we decided that was the best way for it to show that there are multiple versions available in the registry, so users have a choice of which to install.